packages/material-ui-codemod/src/v5.0.0/jss-to-styled.test.js

```diff
@@ -12,215 +12,215 @@ function read(fileName) {
-            {},
-          );
-
-          const expected = read('./jss-to-styled.test/fifth.expected.js');
-          expect(actual).to.equal(expected, 'The transformed version should be correct');
-        });
-
-        it('should be idempotent', () => {
-          const actual = transform(
-            {
-              source: read('./jss-to-styled.test/fifth.expected.js'),
-              path: require.resolve('./jss-to-styled.test/fifth.expected.js'),
-            },
-            { jscodeshift: jscodeshift },
-            {},
-          );
-
-          const expected = read('./jss-to-styled.test/fifth.expected.js');
-          expect(actual).to.equal(expected, 'The transformed version should be correct');
-        });
-      });
-
-      describe('sixth', () => {
-        it('transforms as needed', () => {
-          const actual = transform(
-            {
-              source: read('./jss-to-styled.test/sixth.actual.js'),
-              path: require.resolve('./jss-to-styled.test/sixth.actual.js'),
-            },
-            { jscodeshift: jscodeshift },
-            {},
-          );
-
-          const expected = read('./jss-to-styled.test/sixth.expected.js');
-          expect(actual).to.equal(expected, 'The transformed version should be correct');
-        });
-
-        it('should be idempotent', () => {
-          const actual = transform(
-            {
-              source: read('./jss-to-styled.test/sixth.expected.js'),
-              path: require.resolve('./jss-to-styled.test/sixth.expected.js'),
-            },
-            { jscodeshift: jscodeshift },
-            {},
-          );
-
-          const expected = read('./jss-to-styled.test/sixth.expected.js');
-          expect(actual).to.equal(expected, 'The transformed version should be correct');
-        });
-      });
-
-      describe('seventh', () => {
-        it('transforms as needed', () => {
-          const actual = transform(
-            {
-              source: read('./jss-to-styled.test/seventh.actual.js'),
-              path: require.resolve('./jss-to-styled.test/seventh.actual.js'),
-            },
-            { jscodeshift: jscodeshift },
-            {},
-          );
-
-          const expected = read('./jss-to-styled.test/seventh.expected.js');
-          expect(actual).to.equal(expected, 'The transformed version should be correct');
-        });
-
-        it('should be idempotent', () => {
-          const actual = transform(
-            {
-              source: read('./jss-to-styled.test/seventh.expected.js'),
-              path: require.resolve('./jss-to-styled.test/seventh.expected.js'),
-            },
-            { jscodeshift: jscodeshift },
-            {},
-          );
-
-          const expected = read('./jss-to-styled.test/seventh.expected.js');
-          expect(actual).to.equal(expected, 'The transformed version should be correct');
-        });
-      });
+      // describe('first', () => {
```

Comment:
You can use `describe.skip` and `describe.only` instead (same for `it.skip` and `it.only`).